YoCode/ProcessRunner.cs

```diff
@@ -173,5 +175,17 @@ public void KillCurrentProcessWithChildren()
             FindAndKillChildProcesses(p.Id);
             KillLiveProcess(p);
         }
+
+        public void FindLeftOverProcess()
+        {
+            try
+            {
+                if (!Process.GetProcessById(Pid).HasExited)
+                {
+                    Console.WriteLine("Left Over Process Detected");
```

Comment:
It would be nice to say what the PID is to help people close the right one if this happens. If you're not using Process Explorer it can be hard to tell all the `dotnet` processes apart.